Assets/Scripts/Audio/RTPCTrigger.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/// <summary>
+/// Adjust RTPC of audio when player enters trigger areas
+/// </summary>
+public class RTPCTrigger : MonoBehaviour {
+    [SerializeField]
+    private string rtpcName;
+    [SerializeField]
+    private float rtpcEnterValue;
+    [SerializeField]
+    private float rtpcExitValue;
+
+    private void OnTriggerEnter2D(Collider2D collision) {
+		Debug.Log ("Enter");
```

Comment:
Please remove debug statements.

Reply:
I have not made any changes to the scene other than the trigger guard dlg. Will triggers be created for the ladder and the blood puddle? If so, then I'm assuming that we will be able to attach the RTPCTrigger script to those trigger areas.